src/javascript/app/pages/trade/purchase.js

```diff
@@ -219,16 +226,39 @@ const Purchase = (() => {
                 }
             } });
         }
-        const replace = d => `<strong>${d}</strong>`;
+
+        const is_tick_high = /^tickhigh$/i.test(purchase_data.echo_req.passthrough.contract_type);
+        const is_tick_low  = /^ticklow$/i.test(purchase_data.echo_req.passthrough.contract_type);
```

Comment:
maybe store `purchase_data.echo_req.passthrough.contract_type` in a variable since we're using it more than once?